phab.py

```diff
@@ -129,13 +129,15 @@ def _make_request(self, endpoint, parameters_dict):
             logging.debug("Waiting for {} seconds before making the next request to Conduit.".format(wait_time))  # noqa: E501
             sleep(wait_time)
         parameters = self._to_phab_parameters(parameters_dict)
-        parameters["api.token"] = self._config["api_token"]
+        # Add placeholder API token to not reveal the real one in logs.
+        parameters["api.token"] = "api-..."
```

Comment:
Rather than editing the token in the real parameter-dict would it be safer to work on a modified parameter dict? I.e. something like the following:
```
        safe_parameters = parameters.copy()
        safe_parameters["api.token"] = "api-..."
        logging.debug(
            "POST to Phabricator API on {}: {}".format(
                self._config["api_url"],
                safe_parameters
            )
        )
```

Reply:
Done.